components/inputs/input-checkbox.js

```diff
@@ -63,6 +63,7 @@ export const checkboxStyles = css`
  * A component that can be used to show a checkbox and optional visible label.
  * @slot - Checkbox information (e.g., text)
  * @slot inline-help - Help text that will appear below the input. Use this only when other helpful cues are not sufficient, such as a carefully-worded label.
+ * @slot supporting - Supporting information which will appear below and be aligned with the checkbox.
```

Comment:
I can't help but wonder if consumers should be using the `inline-help` slot instead. Important differences being that inline help is wired up using Aria description so it is limited to text content (not appropriate for other semantics, for example text including links, etc), and gets announced when the user focuses the input. If we support both, I am worried consumers might not realize the difference.

Reply:
I think the main use case (after looking through uses) is the one I put in the video description of the PR, where there's related inputs that need to be "nested" under the checkbox, or displayed when the checkbox is checked.